test-browser/tests/gist.js

```diff
@@ -1,4 +1,6 @@
 'use strict'
+require('dotenv').config()
```

Comment:
can this be added to `init.js` so we are sure `.env` is applied for all the tests, cause we might need env variable for other cases.

Reply:
Okay.